definitions/npm/mozlog_v2.2.x/flow_v0.84.x-/mozlog_v2.2.x.js

```diff
@@ -0,0 +1,33 @@
+declare module 'mozlog' {
+  declare type LoggerFunction = (string, mixed) => void;
+
+  declare interface Logger {
+    trace: LoggerFunction;
+    verbose: LoggerFunction;
+    debug: LoggerFunction;
+    info: LoggerFunction;
+    warn: LoggerFunction;
+    error: LoggerFunction;
+    critical: LoggerFunction;
+
+    // aliases
+    warning: LoggerFunction;
+    o_O: LoggerFunction; // alias to warn
+    O_O: LoggerFunction; // alias to error
+  }
+
+  declare type LoggerCreator = (string) => Logger;
+
+  declare export default function mozlog(options: string | {|
```

Comment:
To be honest I don't know if this is "legit" to use the es module style of exporting if the module itself is using the traditional `exports` object. I did that here so that the types don't leak outside of the libdef.

Reply:
It's fine in most cases, and even preferred.   
The commonjs form can make certain types of API difficult if not impossible to model in Flow.

Also there's the argument that Flow users should most likely be using transpilation, so ES modules...